doc/releases/changelog-0.11.0.md

```diff
@@ -23,106 +26,104 @@
-        @qml.qnode(dev)
-        def circuit():
-            qml.H(0)
-            qml.S(1)
-            qml.T(0)
-            qml.CNOT([0, 1])
-            m1 = catalyst.measure(wires=0)
-            m2 = catalyst.measure(wires=1)
-            return m1, m2
-        circuit()
-    ```
-
-    The PPRs and PPMs are currently only represented symbolically. However, these operations are not yet executable on any backend since they exist purely as intermediate representations for analysis and potential future execution when a suitable backend is available.
-
-    Example MLIR Representation:
-    ```mlir
-      . . .
-        %0 = quantum.alloc( 2) : !quantum.reg
-        %1 = quantum.extract %0[ 1] : !quantum.reg -> !quantum.bit
-        %2 = qec.ppr ["Z"](4) %1 : !quantum.bit
-        %3 = quantum.extract %0[ 0] : !quantum.reg -> !quantum.bit
-        %4 = qec.ppr ["Z"](4) %3 : !quantum.bit
-        %5 = qec.ppr ["X"](4) %4 : !quantum.bit
-        %6 = qec.ppr ["Z"](4) %5 : !quantum.bit
-        %7 = qec.ppr ["Z"](8) %6 : !quantum.bit
-        %8:2 = qec.ppr ["Z", "X"](4) %7, %2 : !quantum.bit, !quantum.bit
-        %9 = qec.ppr ["Z"](-4) %8#0 : !quantum.bit
-        %10 = qec.ppr ["X"](-4) %8#1 : !quantum.bit
-        %mres, %out_qubits = qec.ppm ["Z"] %9 : !quantum.bit
-        %mres_0, %out_qubits_1 = qec.ppm ["Z"] %10 : !quantum.bit
-      . . .
-    ```
-
-* Commuting Clifford Pauli Product Rotation (PPR) operations to the end of a circuit, past non-Clifford PPRs, is now available through the :func:`~.catalyst.passes.commute_ppr` pass transform.
   [(#1563)](https://github.com/PennyLaneAI/catalyst/pull/1563)
+  [(#1564)](https://github.com/PennyLaneAI/catalyst/pull/1564)
+  [(#1577)](https://github.com/PennyLaneAI/catalyst/pull/1577)
   
-  A PPR is a rotation gate of the form :math:`\exp{iP \theta}`, where :math:`P` is a Pauli word (a product of Pauli operators). Clifford PPRs refer to PPRs with :math:`\theta = \tfrac{\pi}{4}`, while non-Clifford PPRs have :math:`\theta = \tfrac{\pi}{8}`.
+  These new compilation passes are currently only represented symbolically and are not yet executable 
+  on any backend. They exist purely as intermediate representations for analysis and potential future 
+  execution when a suitable backend is available.
+
+  The following new compilation passes have been added, and can be accessed from the :mod:`~.passes` 
+  module or in :func:`~.pipeline`:
+
+  * :func:`catalyst.passes.to_ppr <~.passes.to_ppr>`: Clifford + T gates are converted into Pauli product 
+    rotations (PPRs) (:math:`\exp{iP \theta}`, where :math:`P` is a Pauli word (a product of Pauli operators)):
+    * `H` gate → :math:`P = ZXZ` and :math:`\theta = \tfrac{\pi}{4}` 
+    * `S` gate → :math:`P = Z` and :math:`\theta = \tfrac{\pi}{4}` 
+    * `T` gate → :math:`P = Z` and :math:`\theta = \tfrac{\pi}{8}` 
+    * `CNOT` gate → :math:`P = (Z \otimes X)(-Z \otimes \mathbb{1})(-\mathbb{1} \otimes X)` and :math:`\theta = \tfrac{\pi}{4}` 
```

Comment:
These equations are not correct, as they imply H and CNOT can be implemented as a single rotation (exponential). Instead H and CNOT need three rotations (rather than one rotation of a triple product):
```suggestion
    * `H` gate → 3 rotations with :math:`P_1 = Z, P_2 = X, P_3 = Z` and :math:`\theta = \tfrac{\pi}{4}` 
    * `S` gate → 1 rotation with :math:`P = Z` and :math:`\theta = \tfrac{\pi}{4}` 
    * `T` gate → 1 rotation with :math:`P = Z` and :math:`\theta = \tfrac{\pi}{8}` 
    * `CNOT` gate → 3 rotations with :math:`P_1 = (Z \otimes X), P_2 = (-Z \otimes \mathbb{1}), P_3 = (-\mathbb{1} \otimes X)` and :math:`\theta = \tfrac{\pi}{4}` 
```